pkg/kubelet/config/file.go

```diff
@@ -200,3 +201,21 @@ func (s *sourceFile) replaceStore(pods ...*v1.Pod) (err error) {
 	}
 	return s.store.Replace(objs, "")
 }
+
+func checkPlainTextFile(file string) (bool, error) {
+	out, err := exec.Command("file", "--mime-type", file).CombinedOutput()
```

Comment:
Thanks for the suggestion! But most of hosts might not have `mimetype` by default, so the check will fail always.